Formula/cryfs.rb

```diff
@@ -47,6 +48,20 @@ def install
 
   test do
     ENV["CRYFS_FRONTEND"] = "noninteractive"
+
+    # Test showing help page
     assert_match "CryFS", shell_output("#{bin}/cryfs", 10)
+
+    # Test mounting a filesystem. This command will ultimately fail because homebrew tests
+    # don't have the required permissions to mount fuse filesystems, but before that
+    # it should display "Mounting filesystem". If that doesn't happen, there's something
+    # wrong. For example there was an ABI incompatibility issue between the crypto++ version
+    # the cryfs bottle was compiled with and the crypto++ library installed by homebrew to.
+    Dir.mktmpdir do |basedir|
+      Dir.mktmpdir do |mountdir|
+        assert_match "Mounting filesystem", pipe_output("#{bin}/cryfs -f \"#{basedir}\" \"#{mountdir}\"", "password")
```

Comment:
- Can use `2>&1` to avoid printing error messages. 
- Can use single quotes to avoid needing to escape the `"`